index.html

```diff
@@ -31,12 +31,9 @@ <h3>Next meeting</h3>
       <p>
         <b>Topics:</b>
         <p>
-            <a href="https://github.com/blegat">Benoît</a> will give a talk about things related to <a href="https://github.com/JuliaOpt/JuMP.jl">JuMP</a>.
+            <a href="https://github.com/BeastyBlacksmith">Simon</a> will explain how bacteria find their center and shows how to match experiments with julia.
         </p>
-        <p>
-                <a href="https://github.com/mforets">Marcelo</a> will give a short presentation of the github org <a href="https://juliareach.github.io/JuliaReach-website/">JuliaReach</a> (<a href="https://nbviewer.jupyter.org/github/mforets/escritoire/blob/master/talks/JuliaMeetup_Berlin_2019/Talk_Berlin_Julia_Users_Group_2019.slides.html#/">slides here</a>).
-        </p>
-        <b>Date/time:</b> Wednesday, August 7, 2019, 18:30</br>
+        <b>Date/time:</b> Wednesday, September 10, 2019, 18:30</br>
```

Comment:
Should it be *Tuesday*, September 10, or are we moving to Wednesdays now?

Reply:
In either case, can you host, @SimonDanisch ?

Reply:
You are right, I suppose we will go back to Tuesday, but needs to be confirmed before merge

Reply:
Yes I can :)